File: Iniciante/1035/1035.cs
using System; 

class URI 
{
    static void Main(string[] args) 
    {
        string[] linha = Console.ReadLine().Split(' ');
        int a,b,c,d;
        a = Convert.ToInt32(linha[0]);
        b = Convert.ToInt32(linha[1]);
        c = Convert.ToInt32(linha[2]);
        d = Convert.ToInt32(linha[3]);
        if((b>c) && (d>a) && (c+d>a+b) && (c>0) && (d>0) && ( a%2 == 0)) Console.WriteLine("Valores aceitos");
        else Console.WriteLine("Valores nao aceitos");
    }

}

File: Iniciante/1035/1035.java
import java.util.Scanner;


public class Main {
    public static void main(String[] args) {
        
        Scanner i = new Scanner(System.in);
        String valores[] = i.nextLine().split(" ");
        int a = Integer.parseInt(valores[0]);
        int b = Integer.parseInt(valores[1]);
        int c = Integer.parseInt(valores[2]);
        int d = Integer.parseInt(valores[3]);
        if (b>c && d>a && (a+b<c+d) && c>=0 && d>=0 && a%2==0){
            System.out.println("Valores aceitos");
        } else{
            System.out.println("Valores nao aceitos");
        }
        i.close();
    }
}
